Justfile

```diff
@@ -150,6 +150,5 @@ test-all *ARGS: devenv build-cohort-extractor
         --cov=tests \
         --cov-report=html \
         --cov-report=term-missing:skip-covered \
-        tests \
```

Comment:
I've changed it now so that "tests" is the default ARGS value - so it'll do what it did before by default, but still let you run just specific test(s) if you want to.

Reply:
Simon just hit the same thing in https://github.com/opensafely-core/cohort-extractor-v2/pull/278 and solved with adding this path to the config in pyproject.toml.  I think I'm very gently leaning towards the config option, only because it fits with our general ethos of default config in pyproject.toml, happy to be persuaded otherwise though!

Reply:
I don't have a strong feeling either way, as long as I can run individual tests locally :)